src/countItems.js

```diff
@@ -12,11 +12,24 @@
  *  string in the input array
  */
 function countItems(array) {
-  // This is your job. :)
+  let count = 0
+  let dict = {};
+  for (let item of array) {
+    if (item in dict) {
+      dict[item] = dict[item] + 1
+    }
+    else {
+      dict[item] = 1
+    }
+  }
+  console.log(dict)
+  return dict
 }
-
+//My sanity checks aren't working by my code is right so idk why???
```

Comment:
You can't compare object literals using `===`. Well, you can, but it only tells you if they are the same instance of the object, not if they contain the same values.

```javascript
let foo = { apples: 10 };
let bar = { apples: 10 };

console.log(foo === foo); // true
console.log(foo === bar); // false
```